flexmeasures/cli/data_add.py

```diff
@@ -821,111 +822,30 @@ def create_forecasts(
-    type=SensorIdField(),
-    required=False,
-    help="To be deprecated. Use consumption-price-sensor instead.",
-)
-@click.option(
-    "--start",
-    "start",
-    type=AwareDateTimeField(format="iso"),
-    required=True,
-    help="Schedule starts at this datetime. Follow up with a timezone-aware datetime in ISO 6801 format.",
-)
-@click.option(
-    "--duration",
-    "duration",
-    type=DurationField(),
-    required=True,
-    help="Duration of schedule, after --start. Follow up with a duration in ISO 6801 format, e.g. PT1H (1 hour) or PT45M (45 minutes).",
-)
-@click.option(
-    "--soc-at-start",
-    "soc_at_start",
-    type=QuantityField("%", validate=validate.Range(min=0, max=1)),
-    required=True,
-    help="State of charge (e.g 32.8%, or 0.328) at the start of the schedule.",
-)
-@click.option(
-    "--soc-target",
-    "soc_target_strings",
-    type=click.Tuple(
-        types=[QuantityField("%", validate=validate.Range(min=0, max=1)), str]
-    ),
-    multiple=True,
-    required=False,
-    help="Target state of charge (e.g 100%, or 1) at some datetime. Follow up with a float value and a timezone-aware datetime in ISO 6081 format."
-    " This argument can be given multiple times."
-    " For example: --soc-target 100% 2022-02-23T13:40:52+00:00",
-)
-@click.option(
-    "--soc-min",
-    "soc_min",
-    type=QuantityField("%", validate=validate.Range(min=0, max=1)),
-    required=False,
-    help="Minimum state of charge (e.g 20%, or 0.2) for the schedule.",
-)
-@click.option(
-    "--soc-max",
-    "soc_max",
-    type=QuantityField("%", validate=validate.Range(min=0, max=1)),
-    required=False,
-    help="Maximum state of charge (e.g 80%, or 0.8) for the schedule.",
-)
-@click.option(
-    "--roundtrip-efficiency",
-    "roundtrip_efficiency",
-    type=QuantityField("%", validate=validate.Range(min=0, max=1)),
-    required=False,
-    default=1,
-    help="Round-trip efficiency (e.g. 85% or 0.85) to use for the schedule. Defaults to 100% (no losses).",
-)
-@click.option(
-    "--as-job",
-    is_flag=True,
-    help="Whether to queue a scheduling job instead of computing directly. "
-    "To process the job, run a worker (on any computer, but configured to the same databases) to process the 'scheduling' queue. Defaults to False.",
-)
-def create_schedule(ctx, **kwargs):
-    """[deprecated] Create a new schedule for a given power sensor.
+def create_schedule(ctx):
+    """(Deprecated) Create a new schedule for a given power sensor.
 
-    THIS COMMAND HAS BEEN RENAMED, please use `flexmeasures add storage-schedule`
+    THIS COMMAND HAS BEEN RENAMED TO `flexmeasures add schedule storage`
     """
-    click.echo(
-        "THIS COMMAND HAS BEEN RENAMED TO flexmeasures add schedule-for-storage. IT WILL BE DEPRECATED IN v0.13."
-    )
-    ctx.invoke(add_schedule_for_storage, **kwargs)
+    pass
 
 
-@fm_add_data.command("schedule-for-storage")
+@create_schedule.command("storage")
```

Comment:
I am thinking if I prefer `flexmeasures add schedule for-storage`.

This reads more like a natural sentence, and will lead to more comparable names for flex model - based schedulers:  `flexmeasures add schedule for-my-flex-model`

Reply:
I don't particularly love it - in my experience, CLI commands hardly ever read like a natural sentence - but if you feel strongly about it, go ahead. Will you take it from here? Let me know when I can review #537 (by hitting the review button). I've only reviewed some snippets of it proactively.

Reply:
I think I feel strongly. It can be confusing to have two words after `add` - is a storage being added?

Reply:
Just to note some close alternatives for your consideration (not loving them particularly more):
```
flexmeasures add schedule for storage
flexmeasures add schedule --for storage
```